### PR TITLE
feat: update format and guide on policy cmd

### PR DESCRIPTION
**Pull request by ravisuhag**

### CODE REVIEW

File: cmd/policy.go [hunk 1]
```diff
@@ -4,33 +4,53 @@ import (
...
 	"github.com/spf13/cobra"
 )
 
-func policiesCommand(c *app.CLIConfig, adapter v1.ProtoAdapter) *cobra.Command {
+//  PolicyCmd is the root command for the policies subcommand.
+func PolicyCmd(c *app.CLIConfig, adapter v1.ProtoAdapter) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "policies",
-		Short: "manage policies",
+		Use:   "policy",
```
Comment (rahmatrhd) [inline cmd/policy.go:21]: any specific reason for changing the command to singular word?

File: cmd/resources.go [hunk 2]
```diff
@@ -41,34 +56,45 @@ func listResourcesCommand(c *app.CLIConfig) *cobra.Command {
 				return err
 			}
 
-			t := getTablePrinter(os.Stdout, []string{"ID", "PROVIDER", "TYPE", "URN", "NAME"})
-			for _, r := range res.GetResources() {
-				t.Append([]string{
+			report := [][]string{}
+
+			resources := res.GetResources()
+			fmt.Printf(" \nShowing %d of %d policies\n \n", len(resources), len(resources))
```
Comment (rahmatrhd) [inline cmd/resources.go:62]:
```suggestion
			fmt.Printf(" \nShowing %d of %d resources\n \n", len(resources), len(resources))
```

File: cmd/resources.go [hunk 3]
```diff
@@ -6,28 +6,43 @@ import (
...
-func resourcesCommand(c *app.CLIConfig) *cobra.Command {
+func ResourceCmd(c *app.CLIConfig) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "resources",
-		Short: "manage resources",
+		Use:   "resource",
+		Short: "Manage resources",
+		Example: heredoc.Doc(`
+			$ guardian resource list
+			$ guardian resource metadata --id=1
```
Comment (rahmatrhd) [inline cmd/resources.go:23]:
```suggestion
			$ guardian resource metadata set --id=1 key=value
```

File: cmd/policy.go (hunk 1 above)
Comment (ravisuhag) [reply cmd/policy.go:21]: Makes it easier to use. Checked some other good clis for reference as well. e.g. github etc.

Comment (ravisuhag) [reply cmd/policy.go:21]: @rahmatrhd What do you think?

Comment (rahmatrhd) [reply cmd/policy.go:21]: Yeah, as we discussed, we will use singular word and add the plural as the alias

**Review by rahmatrhd — APPROVED** [review]

**Pull request merged**